Integrations/IntSight/IntSight.yml

```diff
@@ -656,7 +661,7 @@ script:
       description: IP address.
       type: String
     - contextPath: Domain.Name
-      description: 'The domain name. For example, "google.com".'
+      description: The domain name. For example, "google.com".
```

Comment:
Looks like some of the description got the `'`sign removed while others not, any idea why?

Reply:
pycharm does that automatically, doesnt really matter